conf/application.conf

```diff
@@ -260,4 +260,5 @@ play.i18n {
     langs = ["en", "cy"]
 }
 
-amend-returns.enabled = true
\ No newline at end of file
+amend-returns.enabled = true
+new-completion-date-already-used-page-exit-flag.enabled = false
```

Comment:
you don't need this new flag - the behaviour should be based on the amend returns enabled flag